r2-navigator/src/main/java/org/readium/r2/navigator/R2WebView.kt

```diff
@@ -979,7 +979,7 @@ class R2WebView(context: Context, attrs: AttributeSet) : R2BasicWebView(context,
         get() {
             var numPages = 0
             try {
-                numPages = this.computeHorizontalScrollRange() / this.computeHorizontalScrollExtent()
+                numPages = ceil((this.computeHorizontalScrollRange() / this.computeHorizontalScrollExtent()).toDouble()).roundToInt()
```

Comment:
@aferditamuriqi Did you intend to round up the result? The problem is that dividing two `Int`s returns a floored `Int`, so you need to convert one of them to `Double` first.

```suggestion
                numPages = ceil(this.computeHorizontalScrollRange().toDouble() / this.computeHorizontalScrollExtent()).toInt()
```

Reply:
Btw I just tried rounding up the result but the layout is worse, so I think the original code was correct.

Reply:
actually no it is correct, this is what i have in all my apps running,

Reply:
did you test if with your suggestion? since that won't work i agree

Reply:
But this line doesn't change anything compared to the original.

* [`computeHorizontalScrollRange()` and `computeHorizontalScrollExtent()`](https://developer.android.com/reference/androidx/core/view/ScrollingView#computeHorizontalScrollRange()) are both returning `Int`
* [dividing two `Int` returns `Int`](https://stackoverflow.com/questions/54106333/how-to-div-of-two-integer-into-decimal-in-kotlin) and is basically like a `floor()`
* from there, `ceil` and `roundToInt` do nothing since the fractional part is 0


<img width="943" alt="Screenshot 2020-10-09 at 08 57 26" src="https://user-images.githubusercontent.com/58686775/95552709-76dd4f80-0a0d-11eb-8d1e-5a81a61eab54.png">